df.history.xml

```diff
@@ -799,27 +799,27 @@
         <enum-item name='SITE_SURRENDERED'/>
         <enum-item name='ENTITY_EXPELS_HF'/>
 
-        <enum-item name='TRADE' since='v0.47.01'/>
```

Comment:
I agree with @PatrikLundell's advice here

Reply:
I agree also.  I'll make that change.